packages/frameworks/src/frameworks.ts

```diff
@@ -18,22 +18,20 @@ const { readdir, readFile, unlink } = promises;
 
 export const frameworks = [
   {
-    name: 'Blitz.js',
-    slug: 'blitzjs',
+    name: 'Blitz.js (Legacy)',
+    slug: 'legacy-blitzjs',
```

Comment:
changing the slug may cause problems for users who already have the blitz preset?

Reply:
Correct, slugs should never change